### PR TITLE
Flight report

### PR DESCRIPTION
**Pull request by klemie**

### CODE REVIEW

File: client/src/components/Module.tsx [hunk 1]
```diff
@@ -115,7 +115,7 @@ const Module: React.FC<ModuleProps> = (props: ModuleProps) => {
 
 	const graph = (
 		<>
-			<RealTimeChart />
+			{/* <RealTimeChart module={mo}/> */}
```
Comment (j7uu) [inline client/src/components/Module.tsx:118]: commented code

File: client/src/components/RealTimeGraph.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	YAxis, 
+	CartesianGrid, 
+	Tooltip, 
+	Legend 
+} from 'recharts';
+import { IModule } from '../utils/entities';
+
+interface GraphProps {
+	dataKeys: string[];
+	// fieldsToRender: string[];
```
Comment (j7uu) [inline client/src/components/RealTimeGraph.tsx:15]: comment 🤔

File: client/src/components/RealTimeGraph.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+const Graph: React.FC<GraphProps> = (props: GraphProps) => {
+	const { realTime, dataKeys, staticData } = props;
+
+	const [data, setData] = useState<any[]>([]);
+
+	useEffect(() => {
+		if (realTime) {
+			//TODO: change to socket connection
```
Comment (j7uu) [inline client/src/components/RealTimeGraph.tsx:27]: todo 👀

File: client/src/components/RealTimeGraph.tsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+		} else {
+			setData(staticData);
+		}
+		console.log('Data:', data);
+		console.log(dataKeys);
```
Comment (j7uu) [inline client/src/components/RealTimeGraph.tsx:32]: console.log

File: client/src/utils/component-documentation.tsx [hunk 5]
```diff
@@ -1,11 +1,11 @@
 import { DocumentationProps } from "../components/ComponentDocumentation";
 import Header from "../components/Header";
 import Module, { Field } from "../components/Module";
-import RealTimeChart from "../components/Graph";
+import RealTimeChart from "../components/RealTimeGraph";
 
 const header = <Header breadCrumbs={[{ name: "breadCrumb", path: "/", active: false }]}/>;
 const module = <Module visualize={true} title="Module Demo" fields={[{ module: "Module Demo", fieldName: 'Field 1', fieldRange: [0,2], fieldValue: 3 } ]} />;
-const graph = <RealTimeChart />;
+// const graph = <RealTimeChart />;
```
Comment (j7uu) [inline client/src/utils/component-documentation.tsx:8]: comment 🤔🤔

File: client/src/views/flight-report-view.tsx [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+                    if (!component.DataConfigId) return;
+                    const response = await api.getDataConfig(component.DataConfigId)
+                    const data = response.data as IDataConfig
+                    console.log(data);
+                    setDataConfigs((prev) => [...prev, data]);
+                });
+            }
+        };
+        getDataConfigs();
+        console.log(context.rocket.Components);
```
Comment (j7uu) [inline client/src/views/flight-report-view.tsx:39]: console.log 🤤 (thanks jack)

File: client/src/views/flight-report-view.tsx [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+			<Grid container>
+				<Header breadCrumbs={breadCrumbs} />
+			</Grid>
+            <Grid item >
+                <Typography variant='h4'>{missionData?.Name} Flight Report</Typography>
+            </Grid>
+            <Grid container style={{ height: '100vh', overflowY: 'scroll' }}>
+                {dataConfigs.map((dataConfig: IDataConfig) => {
+                    return dataConfig.Modules.map((module) => {
+                        // return module.Name;
```
Comment (j7uu) [inline client/src/views/flight-report-view.tsx:61]: yes? no?

**Review by j7uu — APPROVED** [review]
yeee

File: client/src/components/RealTimeGraph.tsx (hunk 3 above)
Comment (klemie) [reply client/src/components/RealTimeGraph.tsx:27]: once telemetry-socket is merged 😶‍🌫️

File: client/src/views/flight-report-view.tsx (hunk 7 above)
Comment (klemie) [reply client/src/views/flight-report-view.tsx:61]: no return lol

**Pull request merged**